cwms-data-api/src/main/java/cwms/cda/data/dao/StreamReachDao.java

```diff
@@ -226,9 +227,9 @@ static StreamReach fromJooqStreamReachRecord(Record record, String stationUnits,
     }
 
     //This should get removed once db is fixed to return bank
-    private Bank getBank(String officeId, String streamId, String streamLocationId) {
+    private Bank getBank(String officeId, String streamId, String streamLocationId, Connection conn) {
         Bank retVal = null;
-        StreamLocation streamLoc = _streamLocationDao.retrieveStreamLocation(officeId, streamId, streamLocationId, "km", "m", "km2");
+        StreamLocation streamLoc = StreamLocationDao.retrieveStreamLocation(officeId, streamId, streamLocationId, "km", "m", "km2", conn);
```

Comment:
Why are these units hard coded?

Reply:
The streamloc retrieval is temporary until the db is fixed to return the bank. This retrieval is just to get the bank, units don't matter.

Reply:
gotcha